apps/judicial-system/backend/src/app/modules/file/guards/limitedAccessViewCaseFile.guard.ts

```diff
@@ -64,7 +63,15 @@ export class LimitedAccessViewCaseFileGuard implements CanActivate {
 
       if (
         indictmentCases.includes(theCase.type) &&
-        caseFile.category === CaseFileCategory.RULING
+        [
```

Comment:
Please update the tests for this guard. They are currently failing because of this change.